src/components/ContactDetails/ContactDetailsAddNewProp.vue

```diff
@@ -139,6 +139,15 @@ export default {
 					property.setParameter('type', defaultData.type)
 				}
 			}
+
+			//We want to wait until the new property is rendered
+			this.$nextTick(() => {
+				let lastAddedElementOfType = document.getElementsByClassName("property--last property-" + id)[0];
```

Comment:
Query selectors are not very Vue-y. Could there be a way to acquire the element via a ref https://vuejs.org/guide/essentials/template-refs.html?

Reply:
We ran into the same issue with https://github.com/nextcloud/contacts/issues/3304 / https://github.com/nextcloud/contacts/pull/3314, and the changes even try to achieve the same goal. https://github.com/nextcloud/contacts/pull/3314 has a wider scope of improvements. Would it be okay to close this PR?

Reply:
FYI https://github.com/vuejs/vue/issues/4225#issuecomment-1199937033 told us that focus selection for custom Vue components is not easy, so we found a tradeoff with component refs and scoped query selectors.